src/main/features/response/index.ts

```diff
@@ -38,7 +38,7 @@ export class Feature {
     app.all(allResponseRoutes, ClaimMiddleware.retrieveByExternalId)
     app.all(/^\/case\/.+\/response\/(?!receipt|summary).*$/, IsDefendantInCaseGuard.check())
     app.all(
-      /^\/case\/.+\/response\/(?!confirmation|full-admission|partial-admission|counter-claim|receipt|summary).*$/,
+      /^\/case\/.+\/response\/(?!confirmation|full-admission|counter-claim|receipt|summary).*$/,
```

Comment:
A `full-admission` should be removed as well since the whole path is being removed.